includes/components/class-admin.php

```diff
@@ -946,6 +949,261 @@ protected function get_extensions_tab() {
+			],
+			[
+				'label' => esc_html__( 'Child theme:', 'hivepress' ),
+				'value' => is_child_theme() ? esc_html__( 'Yes', 'hivepress' ) : esc_html__( 'No', 'hivepress' ),
+			],
+		];
+
+		if ( is_child_theme() ) {
+			$parent_theme = wp_get_theme( $theme->template );
+
+			$theme_info = array_merge(
+				$theme_info,
+				[
+					[
+						'label' => esc_html__( 'Parent Theme Name', 'hivepress' ),
+						'value' => $parent_theme->name,
+					],
+					[
+						'label' => esc_html__( 'Parent Theme Version', 'hivepress' ),
+						'value' => $parent_theme->version,
+					],
+				]
+			);
+		}
+
+		$theme_info[] = [
+			'label' => esc_html__( 'HivePress support:', 'hivepress' ),
+			'value' => current_theme_supports( 'hivepress' ) ? esc_html__( 'Yes', 'hivepress' ) : esc_html__( 'No', 'hivepress' ),
+		];
+
+		// Get report.
+		$report = [
+			'content'  => '',
+			'sections' => [
+				[
+					'label'  => esc_html__( 'WordPress enviroment', 'hivepress' ),
+					'values' => [
+						[
+							'label' => esc_html__( 'HivePress version:', 'hivepress' ),
+							'value' => hivepress()->get_version(),
+						],
+						[
+							'label' => esc_html__( 'Action Scheduler package:', 'hivepress' ),
+							'value' => class_exists( 'ActionScheduler_Versions' ) && class_exists( 'ActionScheduler' ) ? \ActionScheduler_Versions::instance()->latest_version() . ' ' . \ActionScheduler::plugin_path( '' ) : esc_html__( 'No', 'hivepress' ),
+						],
+						[
+							'label' => esc_html__( 'WordPress version:', 'hivepress' ),
+							'value' => get_bloginfo( 'version' ),
+						],
+						[
+							'label' => esc_html__( 'WordPress multisite:', 'hivepress' ),
+							'value' => is_multisite() ? esc_html__( 'Yes', 'hivepress' ) : esc_html__( 'No', 'hivepress' ),
+						],
+						[
+							'label' => esc_html__( 'WordPress memory limit:', 'hivepress' ),
+							'value' => WP_MEMORY_LIMIT,
+						],
+						[
+							'label' => esc_html__( 'WordPress debug mode:', 'hivepress' ),
+							'value' => ( defined( 'WP_DEBUG' ) && WP_DEBUG ) ? esc_html__( 'Yes', 'hivepress' ) : esc_html__( 'No', 'hivepress' ),
+						],
+						[
+							'label' => esc_html__( 'WordPress cron:', 'hivepress' ),
+							'value' => ! ( defined( 'DISABLE_WP_CRON' ) && DISABLE_WP_CRON ) ? esc_html__( 'Yes', 'hivepress' ) : esc_html__( 'No', 'hivepress' ),
+						],
+						[
+							'label' => esc_html__( 'Language:', 'hivepress' ),
+							'value' => get_locale(),
+						],
+						[
+							'label' => esc_html__( 'External object cache:', 'hivepress' ),
+							'value' => wp_using_ext_object_cache() ? esc_html__( 'Yes', 'hivepress' ) : esc_html__( 'No', 'hivepress' ),
+						],
+					],
+				],
+				[
+					'label'  => esc_html__( 'Server environment', 'hivepress' ),
+					'values' => [
+						[
+							'label' => esc_html__( 'Server info:', 'hivepress' ),
+							'value' => isset( $_SERVER['SERVER_SOFTWARE'] ) ? wc_clean( wp_unslash( $_SERVER['SERVER_SOFTWARE'] ) ) : esc_html__( 'No', 'hivepress' ),
```

Comment:
Make sure that there's no other WC-specific code because this one causes a fatal error with WooCommerce disabled